### PR TITLE
Add managed email field for credentialing to Account

### PR DESCRIPTION
**Pull request by MatthewBlanski**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -662,14 +662,6 @@
         <shortDescription>Navigation title for Account Model</shortDescription>
         <value>Account Model</value>
     </labels>
-    <labels>
```
Comment (xinnanlisf) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:665]: Looks like there are tons of label changes here mostly repositionings. Just want to make sure if they are on purpose.

**Review by xinnanlisf — APPROVED** [review]
looks good! very minor questions:

1) Do we want those label changes?
2) Are we adding the new field to the layout?

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (MatthewBlanski) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:665]: They are quite deliberate. We fell out of line with the default ordering in the labels file, so this is restoring that order.

**Comment by MatthewBlanski** [comment]
> looks good! very minor questions:
> 
> 1. Do we want those label changes?
> 2. Are we adding the new field to the layout?

We do want the label readjustment. We should be keeping up with those in each PR, but fell out of line somehow. When I pulled down translations, it reordered them to the MDAPI expected order.

We are deliberately not adding this to a layout as we do not have an Educational Institution layout at this time.

**Comment by spinkelman** [comment]
QA passes - Merging

**Pull request merged**